processing/src/HpsEventFile.cxx

```diff
@@ -38,4 +38,4 @@ bool HpsEventFile::nextEvent() {
   intree_->GetEntry(entry_++);
   
   return true;
-} 
+}  
```

Comment:
There is no need to change the whitespace of HpsEventFile.h or HpsEventFile.cxx in this PR. Can you change them back? If you want to adjust the whitespace, please do this consistently across the whole project in a single PR. Please collaborate with Sarah in doing this, as she has already been thinking about linting in general.

Reply:
I have not intentionally changed the whitespace. I have changed things back. I think Sarah and I will handle that in an iteration on this push request; may be a bit before we have a long time just to talk.